apps/meteor/tests/e2e/page-objects/fragments/home-sidenav.ts

```diff
@@ -64,10 +64,14 @@ export class HomeSidenav {
 		await this.page.locator(`//li[@class="rcx-option"]//div[contains(text(), "${status}")]`).click();
 	}
 
-	async openChat(name: string): Promise<void> {
-		await this.page.locator('role=navigation >> role=button[name=Search]').click();
-		await this.page.locator('role=search >> role=searchbox').type(name);
-		await this.page.locator(`role=search >> role=listbox >> role=link >> text="${name}"`).click();
+	async openChat(name: string, search = true): Promise<void> {
+		if (search) {
+			await this.page.locator('role=navigation >> role=button[name=Search]').click();
+			await this.page.locator('role=search >> role=searchbox').type(name);
+			await this.page.locator(`role=search >> role=listbox >> role=link >> text="${name}"`).click();
+		} else {
+			await this.getSidebarItemByName(name).click();
+		}
```

Comment:
why not call `getSidebarItemByName`directly ?

Reply:
> just keep in mind that you are actually changing the test before people were opening through search and now the test is being performed agains the regular sidebar

That's the objective ;) I want **Omnichannel tests** to get the room directly from the sidebar. This should prevent a problem we are facing where it takes some time for the room to show up in the queue. I created this draft specifically to confirm that.

> why not call `getSidebarItemByName` directly ?

Actually might be a good idea.